molgenis-data-postgresql/src/main/java/org/molgenis/data/postgresql/PostgreSqlQueryGenerator.java

```diff
@@ -773,7 +773,7 @@ private static String getSqlDefaulValue(Attribute attribute, String defaultValue
     switch (attributeType) {
       case BOOL:
         Boolean booleanDefaultValue = (Boolean) defaultTypedValue;
-        sqlDefaultValue = booleanDefaultValue ? "TRUE" : "FALSE";
+        sqlDefaultValue = Boolean.TRUE.equals(booleanDefaultValue) ? "TRUE" : "FALSE";
```

Comment:
booleanDefaultValue == null will now result in sqlDefaultValue == FALSE. Is this the desired behavior?